plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeFileOperations.java

```diff
@@ -228,14 +228,14 @@ public void testDeleteWholePartition()
         assertFileSystemAccesses(
                 "DELETE FROM test_delete_part_key WHERE key = 'p1'",
                 ImmutableMultiset.<FileOperation>builder()
-                        .addCopies(new FileOperation(LAST_CHECKPOINT, "_last_checkpoint", INPUT_FILE_NEW_STREAM), 6) // TODO (https://github.com/trinodb/trino/issues/16782) should be checked once per query
```

Comment:
Why did these go down, fewer times reading the Protocol?

Reply:
The reduction comes from `checkSupportedWriterVersion` change. The method called `TransactionLogAccess.getProtocolEntry` previously.